src/chainstate/stacks/db/mod.rs

```diff
@@ -473,6 +474,17 @@ impl<'a> DerefMut for ChainstateTx<'a> {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct TxStreamData {
```

Comment:
Can you add a rustdoc for this struct?

Reply:
Actually, I think in `develop`, these stream structs became an enum, right? So when you merge, add a rustdoc for the variant.

Reply:
Yup, I refactored this as a `StreamCursor::MempoolTxs` data payload and added rust docs